content/blog/2021-02-12-february-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,172 @@
+this exciting project!
+
+Our search continues for a
+[Developer Advocate](https://weworkremotely.com/remote-jobs/iterative-developer-advocate)
+to support and inspire developers by creating new content like blogs, tutorials,
+and videos - plus lead outreach through meetups and conferences.
+
+Does this sound like you or someone you know? Be in touch!
+
+## Iterative.ai Featured on The New Stack
+
+[Susan Hall](https://thenewstack.io/author/susanhall/) of
+[The New Stack.io](https://thenewstack.io/) interviewed our very own CEO,
+[Dmitry Petrov](https://twitter.com/fullstackml?lang=en), discussing the needs
+of ML engineers and how Iterative.ai makes tools to enable version control and
+CI/CD for versioning data and ML models.
+
+> "ML engineers, they still need collaboration. They need GitHub for
+> collaboration, they need this CI?CD system to resolve [issues] between each
+> other, between the team and productions system." - Dmitry Petrov
+
+<external-link
+href="https://thenewstack.io/iterative-ai-git-based-machine-learning-tools-for-data-engineers/"
+title="Learning Tools for ML Engineers"
+description="Susan Hall"
+link="thenewstack.io"
+image="/uploads/images/2021-02-11/newstack_image.png"/>
+
+## Workshops and Talks
+
+### Developer Advocacy for Data Science
+
+So you saw the post further up. 👆🏽 Curious about developer advocacy or what to
+look for in a hire for this position?
+[Elle O'Brien](https://twitter.com/drelleobrien?lang=en) dove into this recently
+with [Alexey Grigorev](https://twitter.com/Al_Grigor) (author of a
+[Data Science Bookcamp](https://mlbookcamp.com/))
+[in this podcast](https://www.youtube.com/watch?v=jv5W4jXk4P4) on
+[DataTalks.club](http://datatalks.club/) You can watch it here below. 👇🏼
+
+https://www.youtube.com/watch?v=jv5W4jXk4P4
+
+## From the Community
+
+As ever, we have much to share from the great citizens of the DVC community.
+
+### spaCy and DVC Integration
+
+If your NLP team uses spaCy to manage your projects, with spaCy's release of
+v3.0, you can now enjoy DVC integration to manage your workflow like Git! Check
+out the [documentation here](https://spacy.io/usage/projects#integrations) to
+streamline and track your process! 🏆
+
+<external-link
+href="https://spacy.io/usage/projects#integrations/"
+title="spaCy Integration"
+description="spaCy Integration with DVC"
+link="spacy.io"
+image="/uploads/images/2021-02-11/spacy_integration.png"/>
+
+### DagsHub and DVC Integrations
+
+This month two great articles came out regarding the integration of DAGsHub and
+DVC. First, this article:
+[Datasets Should Behave Like Git Repositories](https://dagshub.com/blog/datasets-should-behave-like-git-repositories/)
+walks you through the steps to use DVC in your data versioning. The following
+image shows the dependencies and how you simply need to do a `dvc update` each
+time your dataset or model changes to track the process.
+
+![](/uploads/images/2021-02-11/dags_diagram.png)
```

Comment:
is it a screenshot? this image looks strange a bit ...

Reply:
Yes It's a screenshot from the article.

Reply:
can we pull the image itself? (I don't think + may be add subtitle about its source)?